### ISSUE TITLE
Billmate invoice fee and coupon code

### ISSUE THREAD

**Issue by swnikita** [Billmate/magento#204]
Hello,

I have a question regarding your extension.
Is it possible to exclude Billmate invoice fee from order if 100% coupon code applied? Please, see this screenshot. https://prnt.sc/hdsolw

Thanks!

**Comment by Boxedsolutions** [comment]
No it is not. As the invoice fee is a total, and has nothing to do with discounts. 

Why should you ever need to make an invoice with amount of zero? Give me more information about this.

**Comment by swnikita** [comment]
Hello @Boxedsolutions ,

It was asked by my client, because we need to apply 100% discount on the whole order.
Why is it not possible to disable it? Even shipping methods are available for free (after coupon code appliying).

What do you mean by "invoice fee is a total"? As another custom fields added to the Total block, it is possible to apply the coupon code to them. 

Thanks!

**Comment by Boxedsolutions** [comment]
It was maybe a bit technical, but you dont need to make a payment for a free order. You can activate Magentos Zero Subtotal Checkout, as it is not a payment if the amount is zero

Please look at this documentation about this. 
http://docs.magento.com/m1/ce/user_guide/payment/zero-subtotal-checkout.html

**Comment by swnikita** [comment]
Yes, it was already enabled. Then it works for Admin users. But how we can enable it for FE users? I mean, if the customer has applied 100% coupon code, then no Billmate as a payment method.
